src/pseudopeople/interface.py

```diff
@@ -147,22 +153,22 @@ def generate_decennial_census(
     :param config: An optional override to the default configuration. Can be a path
         to a configuration YAML file or a dictionary.
     :param year: The year (format YYYY) to include in the dataset. Must be a decennial
-        year (e.g. 2020, 2030, 2040). Will return an empty pd.DataFrame if there are no
-        data with this year. If None is provided, data from all years are
+        year (e.g. 2020, 2030, 2040). Will raise a ValueError if there is no data for
```

Comment:
```suggestion
        year (e.g. 2020, 2030, 2040). Will raise a ValueError if there are no data for
```
😈

Reply:
https://www.britannica.com/dictionary/eb/qa/Is-Data-Singular-or-Plural-#:~:text=Answer,in%20visitors%20to%20state%20parks.

Reply:
I am not going to insert myself into such a fraught topic, but we are not being consistent with using it as singular or plural. In the line above we say "if there **is** no data" and in the next sentence we say "data from all years **are** included". We should at least be locally consistent.